client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -20,6 +20,20 @@ import { getUnconnectedSiteUrl } from 'state/selectors';
 import { JETPACK_ONBOARDING_STEPS as STEPS } from '../constants';
 
 class JetpackOnboardingSummaryStep extends React.PureComponent {
+	handleNextStepClick = stepName => () => {
+		const { recordJpoEvent } = this.props;
+		recordJpoEvent( 'calypso_jpo_next_step_clicked', {
+			nextStep: stepName,
+		} );
+	};
+
+	handleCompletedStepClick = stepName => () => {
+		const { recordJpoEvent } = this.props;
+		recordJpoEvent( 'calypso_jpo_completed_step_clicked', {
+			completedStep: stepName,
+		} );
+	};
+
```

Comment:
I'm always late to these parties 😅 What about using only _one_ handler and event, and passing `completed` or `next` as `stepType`?

```js
	handleSummaryStepClick = ( stepName, stepType ) => () => {
		const { recordJpoEvent } = this.props;
		recordJpoEvent( 'calypso_jpo_summary_step_clicked', {
			step: stepName,
			stepType,
		} );
	};
```

Reply:
(I think/hope that might be somewhat close to what you were suggesting in Slack, @AnnaMag)

Reply:
Yes, this is EXACTLY what I meant :)

Reply:
Implemented in #22398.